cmake/modules/Findkeyutils.cmake

```diff
@@ -13,7 +13,7 @@ INCLUDE(CheckCXXSymbolExists)
 
 # include dir
 
-find_path(KEYUTILS_INCLUDE_DIR keyutils.h NO_DEFAULT_PATH PATHS
+find_path(KEYUTILS_INCLUDE_DIR keyutils.h PATHS
```

Comment:
also, `CheckCXXSymbolExists()` is not used, maybe we can take this chance to remove `INCLUDE(CheckCXXSymbolExists)` at line 12 as well.